clients/aws-sdk-transcribe-streaming/src/aws_sdk_transcribe_streaming/client.py

```diff
@@ -0,0 +1,378 @@
+# Code generated by smithy-python-codegen DO NOT EDIT.
+
+from copy import deepcopy
+import logging
+
+from smithy_core.aio.client import ClientCall, RequestPipeline
+from smithy_core.aio.eventstream import DuplexEventStream
+from smithy_core.exceptions import ExpectationNotMetError
+from smithy_core.interceptors import InterceptorChain
+from smithy_core.types import TypedProperties
+from smithy_http.plugins import user_agent_plugin
+
+from .config import Config, Plugin
+from .models import (
+    AudioStream,
+    CallAnalyticsTranscriptResultStream,
+    GET_MEDICAL_SCRIBE_STREAM,
+    GetMedicalScribeStreamInput,
+    GetMedicalScribeStreamOutput,
+    MedicalScribeInputStream,
+    MedicalScribeResultStream,
+    MedicalTranscriptResultStream,
+    START_CALL_ANALYTICS_STREAM_TRANSCRIPTION,
+    START_MEDICAL_SCRIBE_STREAM,
+    START_MEDICAL_STREAM_TRANSCRIPTION,
+    START_STREAM_TRANSCRIPTION,
+    StartCallAnalyticsStreamTranscriptionInput,
+    StartCallAnalyticsStreamTranscriptionOutput,
+    StartMedicalScribeStreamInput,
+    StartMedicalScribeStreamOutput,
+    StartMedicalStreamTranscriptionInput,
+    StartMedicalStreamTranscriptionOutput,
+    StartStreamTranscriptionInput,
+    StartStreamTranscriptionOutput,
+    TranscriptResultStream,
+    _CallAnalyticsTranscriptResultStreamDeserializer,
+    _MedicalScribeResultStreamDeserializer,
+    _MedicalTranscriptResultStreamDeserializer,
+    _TranscriptResultStreamDeserializer,
+)
+from .user_agent import aws_user_agent_plugin
+
+
+logger = logging.getLogger(__name__)
+
+
+class TranscribeStreamingClient:
+    """
+    Amazon Transcribe streaming offers four main types of real-time transcription: **Standard**, **Medical**, **Call Analytics**, and **Health Scribe**.
+
+    * **Standard transcriptions** are the most common option. Refer to for details.
+
+    * **Medical transcriptions** are tailored to medical professionals and incorporate medical terms. A common use case for this service is transcribing doctor-patient dialogue in real time, so doctors can focus on their patient instead of taking notes. Refer to for details.
+
+    * **Call Analytics transcriptions** are designed for use with call center audio on two different channels; if you're looking for insight into customer service calls, use this option. Refer to for details.
+
+    * **HealthScribe transcriptions** are designed to automatically create clinical notes from patient-clinician conversations using generative AI. Refer to [here] for details.
```

Comment:
> Refer to [here] for details.

Hmm, this isn't great either. It looks like this is an issue in the upstream docs. We should make sure the service team is aware.